src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -32,6 +33,31 @@ export interface BreadcrumbProps {
   id?: string;
 }
 
+// Truncate breadcrumb text if beyond 60 characters then add ellipsis at the end.
+const breadcrumbText = (title) => {
+  if (title.length >= 60) {
+    return (
+      <Tooltip content={title} id={"tooltip-" + title}>
```

Comment:
Two things for the id:
1 - use string interpolation 
```
id={`tooltip-${title}`}
```

2 - don't use `title` as part of the id. The `title` is a string with spaces and that doesn't work for the id. 
[id's value must not contain whitespace (spaces, tabs etc.)](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)

Reply:
How to add id here?